<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera</class>
 <widget class="QWidget" name="camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 255, 243);</string>
  </property>
  <widget class="QLabel" name="video_show">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="img_show">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cut">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chepai_result">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;result:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_dj">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>open_dj</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>670</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_dj">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>590</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>send_dj</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>660</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
